### ISSUE TITLE
MS Word 2019 Task Pane Add-in using React with Typescript fails to load in debug mode

### ISSUE THREAD

**Issue by Graph2133** [OfficeDev/generator-office#662]
# Prerequisites
Hi! I am trying to generate new MS Word Task Pane Add-in using template React + Typescript and in debug mode and Add-in is not loaded correctly in TaskPane of MS Word 2019.

# Expected behavior
Add-in must be loaded correctly in MS Word 2019.

# Current behavior
Empty task pane is shown.

## Steps to Reproduce

1) Generate MS Word TaskPane Add-in using React with Typescript template.
2) Run npm start
3) Try to open Add-in in newly opened window of MS Word 2019 Professional.

## Context
* Operating System: Windows 10.
* Node version: 14.17.0.
* Office version: Office 2019 Professional
* Tool version: v1.8.0
* Issue is reproducible on multiple Windows 10 machines for different users
* Add-in works in SharePoint Online Word documents
* The issue happens only after latest update in https://github.com/OfficeDev/Office-Addin-TaskPane-React (webpack was updated to v5+)

I have older version of Add-in with webpack.config and package.json changes before this commit, it uses :
```
     "webpack": "^4.46.0",
    "webpack-cli": "^3.3.12",
    "webpack-dev-server": "^3.11.2"
```
and is created like 2 days before changes in webpack were pushed, it's working fine in MS Word 2019 Proffesional.

## Failure Logs
Error: Office.js has not fully loaded. Your app must call "Office.onReady()" as part of it's loading sequence (or set the "Office.initialize" function). If your app has this functionality, try reloading this page.

## Screenshots
![image](https://user-images.githubusercontent.com/23173060/135093068-5e83f0e9-cc11-4509-be4f-0b864fe9e603.png)

**Comment by millerds** [comment]
Thanks for reporting this.  The commit difference being used by yo office is actually the whole set of changes related to upgrading webpack from v3 to v5 (there was an original change and then some subsequent fixes push to the yo-office branch all at once).  Apparently part of those changes included something that is incompatible with the "IE" version of the web view technology which is what is being use by Office 2019 (not just word).

This is certainly a break on our end and I'm looking into a fix.

**Comment by JensMadsen** [comment]
I experience the same problem with a freshly generated add-in. I tried this because we cannot get our add-in re-certified in App Aource because it suddenly does not work with IE11 engines.

**Comment by JensMadsen** [comment]
I think I actually just solved it on my side adding: `target: ['web', 'es5']` to the webpack config. read here: https://webpack.js.org/migrate/5/#need-to-support-an-older-browser-like-ie-11

**Comment by millerds** [comment]
That also works.

**Comment by sasif2280** [comment]
I also getting the same issue tried above suggestions but nothing works for me 
I have created using Yeoman with react 
I have latest version of node 20.11.1

**Comment by AlexJerabek** [comment]
@sasif2280, if you're unable to open any add-in, could you please open a new issue in the [office-js repo](https://github.com/OfficeDev/office-js/issues)? That will help the right folks from the product team see your issue.
